File: C#/1910. Remove All Occurrences of a Substring.cs
public class Solution {
    public string RemoveOccurrences(string s, string part) {
        StringBuilder sb = new StringBuilder(s);
        int i = s.IndexOf(part);
        while(i != -1){
            sb.Remove(i,part.Length);
            i = sb.ToString().IndexOf(part);
        }
        return sb.ToString();
    }
}

File: Java/1910. Remove All Occurrences of a Substring.java
class Solution {
    public String removeOccurrences(String s, String part) {
        StringBuilder sb = new StringBuilder(s);
        int i = sb.indexOf(part);
        while(i != -1){
            sb.delete(i,i + part.length());
            i = sb.indexOf(part);
        }
        return sb.toString();
    }
}
